Src/Plugins/Security/MySqlConnectionStringValidator.cs

```diff
@@ -15,6 +16,12 @@ internal class MySqlConnectionStringValidator : ValidatorBase
         internal static MySqlConnectionStringValidator Instance;
         internal static IRegex RegexEngine;
 
+        private const string HostRegex = "(?i)(Server\\s*=\\s*(?<host>[\\w\\-.]{3,90}))";
```

Comment:
>\\ [](start = 58, length = 2)

we could use @"" and remove the double slashes

Reply:
Done.

---
In reply to: [570524896](https://github.com/microsoft/sarif-pattern-matcher/pull/165#discussion_r570524896) [](ancestors = 570524896)